google_cloudsql_mysql/main.tf

```diff
@@ -0,0 +1,126 @@
+/**
```

Comment:
something missing here that is used by web sre terraform modules pretty heavily is setting a default application user for the database. is that worth adding in, or a reason to make those resources tf managed elsewhere?

Reply:
do you have a link to an example of how websre did that? Just so i can copypasta instead of thinking.

Reply:
sure - see an example here: https://github.com/mozilla-it/terraform-modules/blob/0905ea5607511f9e57d867953030aa11db7e7989/gcp/database/main.tf#L28